.github/workflows/wasm-publish.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+name: wasm-api
+on:
+  push:
+    tags: 'release-*'
```

Comment:
This is deprecated. We're using 'v2*' now (v20.12.6, etc)

Reply:
Solved